pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1015,6 +1017,18 @@ func setJvmDefault(jvm *api.JvmObject) {
 	}
 }
 
+func checkJvmOnConsole(console *api.ConsoleObject) {
+	if console.Jvm == nil {
+		console.Jvm = &api.JvmObject{}
+	}
+}
+
+func checkJvmOnServer(server *api.KieServerSet) {
```

Comment:
instead of creating these new functions... could we just add this logic here?
https://github.com/kiegroup/kie-cloud-operator/blob/5bae06a7c007fac6ab446c129a4346cdd84b55cd/pkg/controller/kieapp/defaults/defaults.go#L1009-L1018

Reply:
If the jvm is nil we don't know to which object add the empty jvm struct, the setJvmDefault receive just a pointer